web-app/src/components/models.ts

```diff
@@ -8,6 +8,14 @@ export interface Meta {
 }
 export interface IPagination {
   rowsPerPage: number;
+  page: number;
+  rowsNumber?: number;
+}
+export interface IPaginationLabel {
+  totalRowsNumber: number;
+}
+export interface IProps {
+  pagination: IPagination;
 }
```

Comment:
idk, this is random file `models.ts`, okej, we have types here but.. `IProps` is generic name, but its use is very limited